packages/devtools/lib/src/performance/performance_controller.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import '../profiler/cpu_profile_model.dart';
+import '../profiler/cpu_profile_service.dart';
+
+class PerformanceController {
```

Comment:
Seems like a lot of this functionality is shared with the controller for the timeline page. Is it reasonable to create a base class to reduce the amount of duplicate logic? Adding features to both could otherwise get a bit painful.